### ISSUE TITLE
DESIGN - As a user, I want to be able to update my profile header, to further customize my profile 'when I look in the mirror'.

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#3912]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a user, I want to be able to update my profile header, to further customize my profile 'when I look in the mirror'.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:
Further allowing users to customize their profile and MAKE IT LOOK GOOD will allow them to think of their gitcoin profile as their web3 job hunting identity, further increasing their stickiness on the platform.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*:

### Current Behavior
[comment]: # (Describe what actually happened.)
No profile header upload

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Profile header upload will built via https://github.com/gitcoinco/web/issues/3907

### Bounty

Your task for this bounty is to design a profile header background to replace on the Gitcoin profile
.
![header-bg](https://user-images.githubusercontent.com/513929/53929982-e5497700-404c-11e9-8393-dece0b196c98.png)

Here is what a Gitcoin profile looks like: 

<img width="690" alt="Screen Shot 2019-03-12 at 9 50 11 AM" src="https://user-images.githubusercontent.com/513929/54214668-45587700-44ac-11e9-80c6-89c217767a8f.png">

I will pay 20 to 50 DAI per piece of header artwork at my discretion.  I have budget to pay for a dozen of these at 50 DAI each.

Acceptance Criteria
* Please post a link to your design portfolio in your application.
* Please DO NOT start work until I approve you as a contributor.
* Designs must be on brand ( see https://gitcoin.co/presskit ) 
* Feel free to use assets from our creative repo: https://github.com/gitcoinco/creative/
* Expect 1-3 iterations on your work.
* Submit your work as pull request to [this folder on the creative repo](https://github.com/gitcoinco/creative/tree/master/Brand/Wallpaper).  Please submit a (1) svg of the artwork and (2) also a png of the artwork.  Please also (3) submit a screenshot of what the profile will look like with this background image.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 2.0 ETH (275.86 USD @ $137.93/ETH)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3912/2547).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/3912/2547).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $42,588.83 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 12 months from now.
Please review their action plans below:


**1) [charlie4biz](https://gitcoin.co/profile/charlie4biz) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/3912/2547?mutate_worker_action=approve&worker=charlie4biz) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/3912/2547?mutate_worker_action=reject&worker=charlie4biz))_.**

https://drive.google.com/open?id=1l9jK_xh1_dXPMda4F_Kez0t_wJmmCb9I
I have gone through the press kit and waiting on ya approval as a contributor.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3912/2547).

**Comment by owocki** [comment]
@maxstalker nice to meet you.  do you have a portfolio i can check out?

**Comment by owocki** [comment]
> I've found out that original image size is 3000x1100, new images shall be in the same size?

Yes

**Comment by MaxStalker** [comment]
@owocki Sure thing! You can find some stuff here https://www.deviantart.com/krigh/gallery/ and here https://dribbble.com/MaxStalker :)

**Comment by owocki** [comment]
everyone who has applied for this task has a really nice portfolio.  i'm going to approve all 3 of you to start work on ONE of these:

> I will pay 20 to 50 DAI per piece of header artwork at my discretion. I have budget to pay for a dozen of these at 50 DAI each.

**Comment by charlie4biz** [comment]
@owocki any particular format save the banner image?

**Comment by dvdptr** [comment]
@owocki I'd like to go with a minimalistic version, making use of brand colors that are shown as a gradient. first, I played around with some overlapping gradients, but this gone way too wild. that's why I'm going with this minimalistic approach, that I really like. pls let me know what you think.
in case you agree, I'm going to submit a PR. thanks!
<img width="955" alt="gradient02-preview" src="https://user-images.githubusercontent.com/41017948/54135001-47252a80-4419-11e9-840c-ba178670897e.png">

**Comment by owocki** [comment]
@charlie4biz png pls!

@dvdptr that looks good to me.  @PixelantDesign what do you think?

**Comment by charlie4biz** [comment]
![image-sample-3](https://user-images.githubusercontent.com/10086785/54157522-c120d800-4448-11e9-967c-701405799699.png)
![banner-sample-3](https://user-images.githubusercontent.com/10086785/54157540-ca11a980-4448-11e9-8add-276b1703e0e4.png)
What do you think? @owocki @PixelantDesign

**Comment by charlie4biz** [comment]
![image-sample-1](https://user-images.githubusercontent.com/10086785/54157686-1ceb6100-4449-11e9-8c36-412597bfc8a9.png)
![banner-sample-1](https://user-images.githubusercontent.com/10086785/54157699-25439c00-4449-11e9-9914-7d386996a0ba.png)
Just a couple of simple design not to be too distracting. @owocki

**Comment by charlie4biz** [comment]
![image-sample-4](https://user-images.githubusercontent.com/10086785/54160299-73f43480-444f-11e9-8689-ef15a301536a.png)
![image-sample-4-2](https://user-images.githubusercontent.com/10086785/54160300-748ccb00-444f-11e9-804e-08b16b08d8ef.png)
![banner-sample-4](https://user-images.githubusercontent.com/10086785/54160302-748ccb00-444f-11e9-9291-c5263223a37e.png)
What do you think about this final one? My banner design concepts are something simple that won't be distracting or too noisy.

**Comment by owocki** [comment]
i really like the blue one and the top pink one is good too

**Comment by charlie4biz** [comment]
![image-sample-6](https://user-images.githubusercontent.com/10086785/54168120-533ad780-446d-11e9-8518-9558287f8962.png)
![banner-sample-5](https://user-images.githubusercontent.com/10086785/54168121-53d36e00-446d-11e9-9d0f-7c75085a76e6.png)
![banner-sample-6](https://user-images.githubusercontent.com/10086785/54168122-53d36e00-446d-11e9-9a30-689384e7ea31.png)
Which of these other two banners make the cut? :-)

**Comment by owocki** [comment]
i like this one https://user-images.githubusercontent.com/10086785/54168121-53d36e00-446d-11e9-9d0f-7c75085a76e6.png !

**Comment by charlie4biz** [comment]
@owocki ok. You've picked 3 out of all the ones I have sent in. should I discard the others? If there are no corrections, can I PR the ones you picked?

**Comment by owocki** [comment]
yes pls PR the ones I picked :)  

the only change needed (and this is my fault, sorry) is that it now looks like the profile picture is going to be on the left hand side.  we should just make sure they look okay in this format:

<img width="690" alt="Screen Shot 2019-03-12 at 9 50 11 AM" src="https://user-images.githubusercontent.com/513929/54214668-45587700-44ac-11e9-80c6-89c217767a8f.png">

i updated the main ticket desc to reflect this!

**Comment by charlie4biz** [comment]
@owocki alright. Done the required adjustment and submitted 4 header banners you picked. Submitted the two pick colored banner design.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work for 2.0 ETH (266.26 USD @ $133.13/ETH) has been submitted by__:

 1. [@charlie4biz](https://gitcoin.co/profile/charlie4biz)



@owocki please take a look at the submitted work:
* [PR](https://github.com/gitcoinco/creative/pull/55) by @charlie4biz

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3912/2547)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/3912/2547).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $46,304.81 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by MaxStalker** [comment]
@owocki, please take a look at my try. I was trying to stay within brand guidelines and established style, possible theme adjustments and animations. Let me know what you think.
![cannon-yellow-pink](https://user-images.githubusercontent.com/3136647/54238797-7d060400-452a-11e9-9041-87d73ccd77c4.png)
![cannon-blue-yellow](https://user-images.githubusercontent.com/3136647/54238798-7d060400-452a-11e9-9919-5e4de22b0bee.png)
![cannon-green-blue](https://user-images.githubusercontent.com/3136647/54238800-7d060400-452a-11e9-8cb2-5dd72b574a9b.png)

Below you can find previews on how it looks in UI:
![example-cannon-green-blue](https://user-images.githubusercontent.com/3136647/54238716-45975780-452a-11e9-9f56-41580904c048.jpg)
![example-cannon-yellow-pink](https://user-images.githubusercontent.com/3136647/54238717-45975780-452a-11e9-8b4c-398b4376db1f.jpg)
![example-cannon-blue-yellow](https://user-images.githubusercontent.com/3136647/54238718-462fee00-452a-11e9-890f-c7c20f8b3740.jpg)

**Comment by owocki** [comment]
I like a lot!  @PixelantDesign what do you think?

**Comment by MaxStalker** [comment]
...and another one @owocki
![burst-blue](https://user-images.githubusercontent.com/3136647/54243236-b9406100-4538-11e9-956a-d7a6626b0e08.png)
![burst-pink](https://user-images.githubusercontent.com/3136647/54243234-b9406100-4538-11e9-9168-bc8ba1c75f73.png)
![burst-yellow](https://user-images.githubusercontent.com/3136647/54243235-b9406100-4538-11e9-9b74-fc89ded77fce.png)

and previews:
![example-burst-blue](https://user-images.githubusercontent.com/3136647/54243267-ce1cf480-4538-11e9-964f-f3fe1bcc2b66.jpg)
![example-burst-pink](https://user-images.githubusercontent.com/3136647/54243268-ce1cf480-4538-11e9-998a-14ae0fb739f0.jpg)
![example-burst-yellow](https://user-images.githubusercontent.com/3136647/54243269-ce1cf480-4538-11e9-82ca-93a27ac8ca12.jpg)

**Comment by MaxStalker** [comment]
And some more exploration :)
![network-blue](https://user-images.githubusercontent.com/3136647/54245013-328f8200-4540-11e9-8340-02230d727a5a.png)
![network-pink](https://user-images.githubusercontent.com/3136647/54245014-328f8200-4540-11e9-9975-4e895e4f510d.png)
![network-yellow](https://user-images.githubusercontent.com/3136647/54245015-328f8200-4540-11e9-84c0-7a9cc42f9003.png)

...with previews:
![example-netowork-blue](https://user-images.githubusercontent.com/3136647/54245024-38856300-4540-11e9-8c98-1c44a8ffbd75.jpg)
![example-netowork-pink](https://user-images.githubusercontent.com/3136647/54245025-391df980-4540-11e9-8294-a24c166d0504.jpg)
![example-netowork-yellow](https://user-images.githubusercontent.com/3136647/54245026-391df980-4540-11e9-92af-5af87b09f995.jpg)

**Comment by MaxStalker** [comment]
Looking forward to your input, @owocki

**Comment by owocki** [comment]
@MaxStalker WOW 😍 , I think these are very cool + very on brand.  Let me check with @PixelantDesign @willsputra and the other designers.

**Comment by cryptonautica** [comment]
Couple of examples using brand assets/colours.
![Header1](https://user-images.githubusercontent.com/47461733/54295132-c0a45080-45b2-11e9-88b5-1a14cfc6ef63.png)
![Header2](https://user-images.githubusercontent.com/47461733/54295133-c0a45080-45b2-11e9-87c3-8b46e9d45ba2.png)
![Header3](https://user-images.githubusercontent.com/47461733/54295134-c0a45080-45b2-11e9-8cfd-9d1b83bbfa31.png)
![Header1-preview](https://user-images.githubusercontent.com/47461733/54295239-eb8ea480-45b2-11e9-9240-541627815899.png)
![Header2-preview](https://user-images.githubusercontent.com/47461733/54295242-eb8ea480-45b2-11e9-8db7-a062f3f00118.png)
![Header3-preview](https://user-images.githubusercontent.com/47461733/54295243-ec273b00-45b2-11e9-992b-b561b22a01bf.png)

**Comment by PixelantDesign** [comment]
Love them all!

**Comment by owocki** [comment]
Yes I'm thrilled with how this bounty is coming out!

Everyone who has done work for this, can you please submit your work as pull request to [this folder on the creative repo](https://github.com/gitcoinco/creative/tree/master/Brand/Wallpaper).  Please submit a (1) svg of the artwork and (2) also a png of the artwork.

From there I can start paying people out.

**Comment by willsputra** [comment]
Wow these are great! 🎉

**Comment by dvdptr** [comment]
this is my PR for two gradient versions of the header – simple and nice ones :)
https://github.com/gitcoinco/creative/pull/59

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.45000  ETH (60.96 USD @ $135.47/ETH) has been granted to @dvdptr for this issue from @owocki. ⚡️ 

Nice work @dvdptr! Your tip has automatically been deposited in the ETH address we have on file.

 * $49022.56 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by dvdptr** [comment]
thanks a lot @owocki 
is the bounty still open for further submissions or do you close after paying? I'd be interested to design some more, let me know what you think.

**Comment by charlie4biz** [comment]
Hi @owocki already done my PR 3 days ago which has been merge. Above https://github.com/gitcoinco/web/issues/3912#issuecomment-472137606

**Comment by owocki** [comment]
thx just tipped everyone who has a merged PR.  if you all submit your images as a PR i can pay you out!

**Comment by dvdptr** [comment]
@owocki 
do you agree that I keep on working and submit another graphic or do you want to close the bounty soon? thanks

**Comment by owocki** [comment]
I'm going to close the bounty for now, as I think we have enough headers to get started

once https://github.com/gitcoinco/web/issues/3907 is implemented i will likely open up a 2nd bounty for more submissions

thank you to everyone for your hard work!

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__This Bounty has been completed.__

Additional Tips for this Bounty:
* owocki tipped 0.4500 ETH worth 60.96 USD to dvdptr.

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3912/2547)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $48,706.64 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by owocki** [comment]
@charlie4biz your image is kind of pixelated.. got a high res?

**Comment by owocki** [comment]
@iamonuwa 
[header_images.zip](https://github.com/gitcoinco/web/files/3254580/header_images.zip)

**Comment by iamonuwa** [comment]
Cool

**Comment by charlie4biz** [comment]
Ok.@owocki done. Having some technical issues attaching. Will send before the day runs out.

**Comment by charlie4biz** [comment]
> @charlie4biz your image is kind of pixelated.. got a high res?

![banner-sample-6](https://user-images.githubusercontent.com/10086785/58998689-03647600-87fa-11e9-965a-eab39d8f0fd1.png)

![polkadot-banner](https://user-images.githubusercontent.com/10086785/58998693-06f7fd00-87fa-11e9-80a0-50fc82b9e380.png)

**Comment by charlie4biz** [comment]
![rocket-banner](https://user-images.githubusercontent.com/10086785/59000616-2cd5cf80-8803-11e9-9fbe-0dcb72b7109c.png)

**Comment by owocki** [comment]
does anyone who submitted images above have vectorized versions of the artwork?

**Comment by MaxStalker** [comment]
@owocki mine were all vector based :)

**Comment by owocki** [comment]
opening this up for a few more subbmisions https://github.com/gitcoinco/web/issues/5145
